onadata/settings/base.py

```diff
@@ -535,6 +535,7 @@ def skip_suspicious_operations(record):
     AZURE_ACCOUNT_KEY = env.str('AZURE_ACCOUNT_KEY')
     AZURE_CONTAINER = env.str('AZURE_CONTAINER')
     AZURE_URL_EXPIRATION_SECS = env.int('AZURE_URL_EXPIRATION_SECS', None)
+    AZURE_OVERWRITE_FILES = True
```

Comment:
corresponding change added to kpi by https://github.com/kobotoolbox/kpi/commit/fa3157d6b53e7069d057c2ae77186dc50ae1608f#diff-fda91693febb37a79e7e40d7252c7851ae75b5e07eded7111299c8a078b9a93eR670